kube-client/src/client/builder.rs

```diff
@@ -158,7 +158,13 @@ where
     #[cfg(feature = "gzip")]
     let stack = ServiceBuilder::new()
         .layer(stack)
-        .layer(tower_http::decompression::DecompressionLayer::new())
+        .layer(
+            tower_http::decompression::DecompressionLayer::new()
+                .no_br()
```

Comment:
intentionally disable anything but GZIP compression - 

reading https://github.com/kubernetes/kubernetes/issues/112296 show there isn't like to be any other option in Kube for a while, but these flags could easily be refitted when/if Kube gains alternatives.

Reply:
agreed. think this is sensible also.